### DISCUSSION TITLE
Log in to the admin panel and return to the blank page in the safari browser of IOS 13.7.

### DISCUSSION THREAD

**Discussion by LittleHans8** [filamentphp/filament#3583] [Help] [answered]
May I ask whether filament is not compatible with systems below IOS 13.7? I tried to use filament in Safari browser of IOS 13.7, and found that after login, all pages returned blank pages.

<img width="404" alt="image" src="https://user-images.githubusercontent.com/15800501/185736631-5e9af621-9a12-4e7e-810b-1c41e33e927a.png">

<img width="404" alt="image" src="https://user-images.githubusercontent.com/15800501/185736635-7e7b6189-cc13-4942-ae05-6ff4325a09e9.png">

**Comment by danharrin** [comment, marked as answer]
Never seen this before, either way it would probably be an issue with Livewire and not Filament
